src/PKSim.Core/Mappers/IndividualToIndividualBuildingBlockMapper.cs

```diff
@@ -50,14 +47,20 @@ public override IndividualBuildingBlock MapFrom(Individual input)
          addOriginDataToBuildingBlock(buildingBlock, PKSimConstants.UI.Weight, input.OriginData.Weight);
          addOriginDataToBuildingBlock(buildingBlock, PKSimConstants.UI.Population, input.OriginData.Population?.DisplayName);
 
-         input.OriginData.AllCalculationMethods().Where(cm => _calculationMethodCategoryRepository.HasMoreThanOneOption(cm, input.Species)).MapAllUsing(_calculationMethodDTOMapper)
-            .Each(x => addOriginDataToBuildingBlock(buildingBlock, x.DisplayName, x.CategoryItem.DisplayName));
+         input.OriginData.AllCalculationMethods().Where(cm => _calculationMethodCategoryRepository.HasMoreThanOneOption(cm, input.Species))
+            .Each(x => addCalculationMethodOriginDataToBuildingBlock(buildingBlock, x));
 
          buildingBlock.OriginData.ValueOrigin = input.OriginData.ValueOrigin.Clone();
 
          return buildingBlock;
       }
 
+      private void addCalculationMethodOriginDataToBuildingBlock(IndividualBuildingBlock buildingBlock, CalculationMethod calculationMethod)
+      {
+         var repInfo = _representationInfoRepository.InfoFor(RepresentationObjectType.CATEGORY, calculationMethod.Category);
```

Comment:
I had to replace the dto mapper with the representationInfoRepository to gather the DisplayName property.
This is what the mapper would do when creating the DTO